File: in_Java/0026 Remove Duplicates from Sorted Array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if(nums == null || nums.length == 0) return 0;
        if(nums.length == 1) return 1;
        int i = 0;
        int j = 0;
        while(j < nums.length){
            while(j < nums.length && nums[i] == nums[j]) j++;
            if(j < nums.length) nums[++i] = nums[j];
        }
        return i+1;
    }
}

File: in_Python/0026 Remove Duplicates from Sorted Array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        i = 0
        j = 0
        idx = 0
        while j < len(nums):
            while j < len(nums) and nums[j] == nums[i]:
                j += 1
            nums[idx] = nums[i]
            i = j
            idx += 1
        return idx